src/amber/router/session/cookie_store.cr

```diff
@@ -0,0 +1,80 @@
+module Amber::Router::Session
+  # This is the default Cookie Store
+  class CookieStore < AbstractStore
+    property secret : String
+    property key : String
+    property expires : Int32
+    property store : Amber::Router::Cookies::Store
+    property session : Hash(String, String)
+
+    def initialize(@store, @key, @expires, @secret)
+      @session = current_session
+      @session[key] = SecureRandom.uuid
+    end
+
+    def id
+      @session[key]
+    end
+
+    def destroy
+      @session.clear
+    end
+
+    def [](key)
```

Comment:
@eliasjpr In Rails `session[:key]` is the same as `session["key"]`. It extends HashWithIndifferentAccess. It super easy to do that here, but just storing it as a string but converting symbols to strings before looking it up. Like we're currently doing here: https://github.com/Amber-Crystal/amber/blob/master/src/amber/router/pipe/session.cr#L74